.test-defs/ShootKubernetesUpdateTest.yaml

```diff
@@ -0,0 +1,24 @@
+kind: TestDefinition
+metadata:
+  name: shoot-kubernetes-update-test
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: Tests the kubernetes update of a shoot.
+
+  activeDeadlineSeconds: 1800
+  labels: []
+
+  config:
+  - name: GO111MODULE
```

Comment:
Can we remove `GO111MODULE` from the test definitions with golang-1.13?
Ref https://golang.org/doc/go1.13#modules